i18n/index.js

```diff
@@ -1,32 +0,0 @@
-import path from 'path';
-import LanguageDetector from 'i18next-browser-languagedetector';
-import NextI18Next from 'next-i18next';
-import { initReactI18next } from 'react-i18next';
-import getConfig from '../tenant.config';
-
-const config = getConfig();
-
-export default new NextI18Next({
-  use: [LanguageDetector, initReactI18next],
-  localePath: path.resolve('./public/static/locales'),
-  defaultLanguage: config.languages[0] ? config.languages[0] : 'en',
-  fallbackLng: config.languages[0] ? config.languages[0] : 'en',
-  debug: false,
-  otherLanguages: config.languages, // list all languages here
-  detection: {
-    // check if language is cached in local storage, otherwise retrieve from browser language
-    order: ['localStorage', 'navigator'],
-
-    // next-i18next by default searches for the 'next-i18next' cookie, local storage key on server requests
-    lookupLocalStorage: 'language',
```

Comment:
Many code is depending on having the language be set in the localStorage variable `language`. Now that it is stored in `i18nextLng`, all this code probably won't work correctly any more defaulting to `en`. We should either store the value of `i18nextLng` also in `language` or (maybe the cleaner way) replace usage of `language` with `i18nextLng` everywhere in our code.